translations/es/LC_MESSAGES/messages.po

```diff
@@ -223,7 +228,7 @@ msgstr ""
 
 #: /content/includes/home.yaml
 msgid "Building the future web, together."
-msgstr "Armando el futuro de la web, juntos."
+msgstr "Juntos, Armando el futuro de la web."
```

Comment:
@SidVal - You still want FiliMX to do some changes, correct?  Once he makes the changes, and you approve (through GitHub's approval mechanism), I'll merge the changes.